payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -170,6 +170,13 @@ data class PaymentMethod internal constructor(
             return code
         }
 
+        fun hasDelayedSettlement() = setOf(
```

Comment:
I think this a great change to make this a flag on the payment method!

Reply:
On `PaymentMethod.Type`, you mean? I think this way is more readable, but wouldn't really argue against the change. @ccen-stripe seems to want to go in the opposite direction. I'll keep like this for now.